### PR TITLE
Tutorial Menu

### PR DESCRIPTION
**Pull request by thorgod**
Added tutorial menu. This splits up the tutorials into stations. Also gives them descriptions of each station before starting each tutorial. 

(Need a little help refactoring some functions on the LUA script so I don't have copied code over and over)

### CODE REVIEW

File: scripts/tutorial_03_weapons.lua [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+--- -Beam weapons fire at a target's hull by default, but the Weapons officer can also target specific subsystems to disable an enemy. If you simply wish to destroy an enemy, however, it's best left on hull.
+---
+--- Shields: The Weapons officer is responsible for activating the ship's shields and modulating their frequency. It might be tempting to keep the shields up at all times, but they drain significantly more power when active. Certain shield frequencies are especially resistant to certain beam frequencies, which can also be detected in targets by the Science officer. Unlike beam weapons, however, remodulating the shields' frequency brings them offline for several seconds and leaves the ship temporarily defenseless.
+-- Type: Basic
+require("utils.lua")
+require("utils_tutorial.lua")
```
Comment (thorgod) [inline scripts/tutorial_03_weapons.lua]: Need to remove this. Going to re-commit

**Comment by thorgod** [comment]
Updated pull request to include minor changes.

**Comment by thorgod** [comment]
Is something wrong with this pull request or you would like changed?

**Comment by daid** [comment]
I haven't found the energy yet to review it. And then it went out of my view for a while. I might find some time tomorrow.

**Comment by thorgod** [comment]
Thanks man. No problem, I completely understand.

**Pull request merged**